Library/Formula/eg.rb

```diff
@@ -0,0 +1,40 @@
+class Eg < Formula
+  homepage "https://github.com/srsudar/eg"
+  url "https://github.com/srsudar/eg/archive/v0.0.3.tar.gz"
+  sha256 "c257e334f2a2e494f9b9791b92c1d3f7b00da3e2e7ec47f02d720342b466f968"
+
+  def install
+    if Dir.exist? "#{var}/examples"
```

Comment:
Ah, completely forgot about uninstalling.

The reason I put it there is so that other formulae can use that directory to install more examples, as it's supposed to be extensible (like the MANPATHs for `man`, other formulae put their manpages in a MANPATH),

If a formula has eg pages and eg is uninstalled, should those formula's pages remain installed, i.e. should the whole examples directory be removed? I don't think so (although of course the built-in eg pages should be, which would be easier if formulae could have an uninstall method (can they?)).

Not sure what the best thing to do about this is.